src/globals/scss/tools/_iff.scss

```diff
@@ -1,5 +1,5 @@
 // Wrapper for Sass' built-in `if` function that does not require you to pass
 // a value for $if-false.
-@function iff($condition, $if-true) {
+@function govuk-iff($condition, $if-true) {
```

Comment:
I wonder if this function is worth keeping given the intention was to have a shorter way of writing

```
if($important, !important, null)
```

and now this is:
```
govuk-iff($important, !important)
```

Happy to address this in a later PR though.

Reply:
happy to keep as `iff`